src/test/kotlin/com/forgerock/uk/openbanking/tests/functional/payment/domestic/vrp/api/v3_1_10/GetDomesticVrpDetails.kt

```diff
@@ -33,6 +33,26 @@ class GetDomesticVrpDetails(
         assertThat(domesticVrpDetails.data.paymentStatus).isNotNull()
     }
 
+    fun getDomesticVrpDetailsWithMultiplePaymentsTest() {
+        // Given
+        val consentRequest = OBDomesticVrpConsentRequestTestDataFactory.aValidOBDomesticVRPConsentRequest()
+        val result1 = createDomesticVrpApi.submitPayment(consentRequest)
+        val result2 = createDomesticVrpApi.submitPayment(consentRequest)
+
+        // When
+        val domesticVrpDetails1 = getDomesticVrpDetails(result1)
+        val domesticVrpDetails2 = getDomesticVrpDetails(result1)
+
+        // Then
+        assertThat(domesticVrpDetails1).isNotNull()
```

Comment:
can we add a sanity test that checks that payment1.domesticVrpId != payment2.domesticVrpId